docs/earthfile/earthfile.md

```diff
@@ -564,6 +583,57 @@ A global `ARG` is an arg that is made available to all targets in the Earthfile.
 
 Global args may only be declared in base targets.
 
+## LET (experimental)
```

Comment:
Fixed. I was leaning back and forth on this one - `LET` is so closely related to `ARG` that I felt it might make sense to keep the two together, even though it's  experimental.

But at least right now, with our docs page limitations, I think you're probably right - it belongs at the end.